docs/_future/migrate_existing_app.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Migrate an existing app
+title: Setup an existing app
 order: 9
-slug: migrate-existing-app
+slug: setup-existing-app
 lang: en
 layout: tutorial
-permalink: /future/migrate-existing-app
+permalink: /future/setup-existing-app
```

Comment:
Could also we add a redirect from `/future/migrate-existing-app` to here so folks don't lose the page if they bookmarked it?

Reply:
Yes! I've updated the existing references to setup and added a redirect_from property! Thanks